composer.json

```diff
@@ -86,122 +86,121 @@
         "sebastian/phpcpd": "2.0.4"
     },
     "replace": {
```

Comment:
Hi @orlangur 

Can you help me by providing me with a correct version of composer.json ?

I'm not sure what version each of the modules are supposed to be in the `replace` section.  All I do know is that `"magento/module-sales": "100.3.0-dev",`

Reply:
And can you confirm my target branch?  My PR is currently targeting `develop` branch, whereas maybe it should be `2.2.0-preview`

Reply:
It should definitely be `develop`, please change back. 

`develop` branch reflects what will be later released as `2.3`, most of the modules are `100.3.0-dev` so I assume your change makes module `Sales` `100.4.0-dev` (@joni-jones, please check).